client/my-sites/patterns/index.tsx

```diff
@@ -0,0 +1,33 @@
+import DocumentHead from 'calypso/components/data/document-head';
+import Main from 'calypso/components/main';
+import { PatternsCategoryBody } from 'calypso/my-sites/patterns/bodies/category';
+import { PatternsHomeBody } from 'calypso/my-sites/patterns/bodies/home';
+import type { PatternGalleryFC } from 'calypso/my-sites/patterns/types';
+
+type PatternsHomePageProps = {
```

Comment:
This should be renamed.
```suggestion
type PatternsIndexProps = {
```